vscode-dotnet-runtime-library/src/Acquisition/InstallTrackerSingleton.ts

```diff
@@ -315,38 +315,38 @@ ${convertedInstalls.map(x => `${JSON.stringify(x.dotnetInstall)} owned by ${x.in
                 );
             }
 
-            this.eventStream.post(new AddTrackingVersions(`Updated ${keyStr} :
+            this.eventStream.post(new AddTrackingVersions(`Updated ${idStr} :
 ${existingVersions.map(x => `${JSON.stringify(x.dotnetInstall)} owned by ${x.installingExtensions.map(owner => owner ?? 'null').join(', ')}\n`)}`));
-            await this.extensionState.update(key, existingVersions);
-        }, keyStr, installObj);
+            await this.extensionState.update(id, existingVersions);
+        }, idStr, installObj);
     }
 
-    public async checkForUnrecordedLocalSDKSuccessfulInstall(context : IAcquisitionWorkerContext, dotnetInstallDirectory: string, installedInstallKeysList: InstallRecord[]): Promise<InstallRecord[]>
+    public async checkForUnrecordedLocalSDKSuccessfulInstall(context : IAcquisitionWorkerContext, dotnetInstallDirectory: string, installedInstallIdsList: InstallRecord[]): Promise<InstallRecord[]>
     {
-        return this.executeWithLock( false, async (dotnetInstallDir: string, installedInstallKeys: InstallRecord[]) =>
+        return this.executeWithLock( false, async (dotnetInstallDir: string, installedInstallIds: InstallRecord[]) =>
         {
-            let localSDKDirectoryKeyIter = '';
+            let localSDKDirectoryidIter = '';
```

Comment:
LocalSDKDirectoryIdIter

Reply:
Thanks for your approval, looks like I forgot to do a case-sensitive search on this file. I also improved some of the names in other spots where it looks like I missed it.

You're right in that I will need to go into the GDPR tool and reclassify the events and update some of our queries as well.